### DISCUSSION TITLE
Map with Image entity instead of URL

### DISCUSSION THREAD

**Discussion by poachedindividual** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#697] [Q&A] [answered]
Is there a way to use an image entity, instead of an image URL, for the map? 

The Deebot 4 Home Assistant integration no longer provides a camera entity for the map, it now uses an image entity. 

https://github.com/DeebotUniverse/Deebot-4-Home-Assistant/releases/tag/2.0.0
Breaking Change #382

**Comment by PiotrMachowski** [comment, marked as answer]
You can just provide image entity instead of camera entity.
